scilpy/io/mti.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+import logging
+import os
+import sys
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.image.volume_math import concatenate
+from scilpy.io.image import load_img
+from scilpy.io.utils import get_acq_parameters
+from scilpy.reconst.mti import adjust_B1_map_intensities, smooth_B1_map, \
+    process_contrast_map
+
+
+def add_common_args_mti(p):
```

Comment:
A really short docstring. Could be only a description to say this defines mti arguments used for these scripts: scil_....